### ISSUE TITLE
error comando /quests 

### ISSUE THREAD

**Issue by superkaito** [PikaMug/Quests#2229]
### What is your Quests version / build number (do *not* say "latest")?

5.0.3

### Describe the problem and how to replicate it.

Español
He tenido un problema desde hace tiempo en el que el comando /quests no funciona. Intenté replicar todos mis complementos en otro servidor y todo funcionaba perfectamente. Sin embargo, en mi servidor, lo único que no funcionaba era /quests, mientras que /quest sí funcionaba.

Al poner los mismos complementos en otro servidor, el comando /quests funcionaba, así que no es una incompatibilidad.

Inglés
I have had an issue for some time where the /quests command does not work. I tried replicating all my plugins on another server and everything worked perfectly. However, on my server, the only thing that didn't work was /quests, while /quest did work.

When I put the same plugins on another server, the /quests command worked, so it's not an incompatibility.

### Any console errors to share?

```shell
Not even a single one
```

**Comment by PikaMug** [comment]
I would need to see your startup log in order to try diagnosing this.

Unless your permissions are the issue, a plugin incompatibility is still most likely.

**Comment by superkaito** [comment]
no the permissions are not the problem I have op

There are no cyst errors in the console although I will send you the logs

**Comment by superkaito** [comment]
[latest.log](https://github.com/user-attachments/files/15929860/latest.log)

**Comment by PikaMug** [comment]
Try updating [WolfyUtilities](https://www.spigotmc.org/resources/wolfyutilities-core-api-1-17-1-1-20-6.64124/); I see pretty significant errors which may affect commands in a roundabout way.

I also see you're using a dev build of MMOItems. I wouldn't recommend downgrading, but also try updating to a newer build from https://phoenixdevt.fr/devbuilds

**Comment by superkaito** [comment]
actualice WolfyUtilities no funciono también lo elimine y tampoco

**Comment by superkaito** [comment]
update WolfyUtilities doesn't work and /quests still doesn't work, also delete it and it doesn't work either

**Comment by PikaMug** [comment]
@superkaito Please let me know if the latest development build still has this issue:
https://ci.codemc.io/job/PikaMug/job/Quests/

**Comment by superkaito** [comment]
I have found the cause of the problem. It was because the command was blocked by a permission in the default group. When I switched groups, everything was resolved.

**Comment by superkaito** [comment]
[10:22:54 WARN]: me.pikamug.quests.exceptions.StageFormatException: Stage cannot be empty, see quest d stage 1
[10:22:54 WARN]:        at Quests-5.1.0.jar//me.pikamug.quests.storage.implementation.file.BukkitQuestYamlStorage.loadQuestStages(BukkitQuestYamlStorage.java:842)
[10:22:54 WARN]:        at Quests-5.1.0.jar//me.pikamug.quests.storage.implementation.file.BukkitQuestYamlStorage.init(BukkitQuestYamlStorage.java:110)
[10:22:54 WARN]:        at Quests-5.1.0.jar//me.pikamug.quests.BukkitQuestsPlugin.lambda$reload$4(BukkitQuestsPlugin.java:717)
[10:22:54 WARN]:        at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:101)
[10:22:54 WARN]:        at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
[10:22:54 WARN]:        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[10:22:54 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
[10:22:54 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
[10:22:54 WARN]:        at java.base/java.lang.Thread.run(Thread.java:1583)
[10:22:54 WARN]: [Quests] Unable to consider custom objectives because quest for 000001 was null


Although now I have another problem. When I create a mission, it gets canceled, and an error appears in the console.

**Comment by PikaMug** [comment]
> I have found the cause of the problem. It was because the command was blocked by a permission in the default group. When I switched groups, everything was resolved.

Glad to hear it!

It looks like your quest 'd' has no stages, but please create a new issue if you need assistance with that.
